app/controllers/admin/orders_controller.rb

```diff
@@ -3,7 +3,23 @@
 module Admin
   class OrdersController < AdminController
     def index
-      render :index, locals: { orders: Order.includes(:employee).all.order(:status) }
+      # render index, locals: { orders: Order.includes(:employee).all.order(:status) }
+      @orders = Order.includes(:employee).all.order(:status)
+
+      # respond_to do |format|
+      #  format.html
+      #  format.csv { send_data @orders.to_csv, filename: "orders-#{Date.today}.csv" }
+      # end
+
+      respond_to do |format|
+        format.html
+        # render :template => 'admin/orders/index.html.erb'
```

Comment:
i think this is not needed here, as we are only rendering csv

Reply:
Of course it was a draft, now there are no unnecessary entries anymore :)